tests/test_questionnaire_validator.py

```diff
@@ -345,9 +339,10 @@ def test_invalid_relationship_no_list_specified():
     validator.validate()
 
     expected_for_list_error = {
-        "message": error_messages.FOR_LIST_NEVER_POPULATED,
-        "list_name": "not-a-list",
-        "block_id": "relationships",
+        "message": ValueSourceValidator.ANSWER_REFERENCE_INVALID,
+        "identifier": "first-name",
+        "json_path": "groups.[0].blocks.[0].question.title.placeholders.[0].transforms.["
+        "0].arguments.list_to_concatenate.[0].identifier",
     }
```

Comment:
To me, this implies the schema isn't correct enough to test this scenario?. The body no longer matches the test name.

Reply:
I wasn't confident on this, you may be correct. As this json was getting used for other tests so i was reluctant to change it. In this case ,perhaps my lack of knowledge in schema.

Reply:
I think the JSON has an invalid ID that needs fixing before getting to the bit that does the actual test this intends.